src/App.js

```diff
@@ -1,53 +1,145 @@
 import { useEffect, useState } from 'react';
 import './App.css';
 import ProfileCard from './components/profile-card';
-import profileData from './mock-data';
-import {message} from 'antd'
+import {message, notification, Spin, Tag, Descriptions, Row} from 'antd'
 import _ from 'lodash'
+import { getProfilesData } from './network' 
+
+function mapProfileData(rawData){
+  return rawData.map(({name, email, dob, picture}) => ({
+    name: name.first + ' ' + name.last,
```

Comment:
how about this, which also doesn't crash if first or last is null

```suggestion
    name: `${name.first} ${name.last}`,
```

Reply:
Nice! I'll update it 👍